openqabot/incrementapprover.py

```diff
@@ -384,9 +384,8 @@ def _make_scheduling_parameters(
             "ARCH": build_info.arch,
             "BUILD": build_info.build,
             "INCREMENT_REPO": config.build_project_url(DOWNLOAD_BASE) + repo_sub_path,
-            "_OBSOLETE": "1",
-            "_ONLY_OBSOLETE_SAME_BUILD": "1",
```

Comment:
How about just referencing it in the definition like before just now with `**OBSOLETE_PARAMS`?

Reply:
done